ui/apps/dotcms-ui/src/app/api/services/dot-contentlet-locker/dot-contentlet-locker.service.ts

```diff
@@ -15,13 +15,13 @@ export class DotContentletLockerService {
      * @returns Observable<any>
      * @memberof PageViewService
      */
-    lock(inode: string): Observable<DotContentletLockResponse> {
+    lock(inode: string): Observable<DotCMSResponse<DotContentletLockResponse>> {
```

Comment:
Did you test that this is working? I mean, beyond that TSC compiles?

Reply:
Yes, its works.